### ISSUE TITLE
What kind of style should this map have?

### ISSUE THREAD

**Issue by RedSilencer** [quat1024/PortalMaps#4]
It's very open and has different floor heights. Obviously, Old Aperture and anything like it is off of the table because of the variety of puzzle elements that exists in this puzzle.

I'm leaning towards a sort of **hybrid Art Therapy / Clean** style to introduce some colors and to provide a better atmosphere, but we could bite the bullet and go for a sort of **retro / classy, warmly-lit** sort of "focused" design that does away with the white noise of decoration.

**Comment by quat1024** [comment]
I was thinking the wall opposite the entrance (assuming it's where it is in pti) could be "knocked out" to make space for some decoration... like, if wheatley style was used, that would be an :ok_hand: spot for the vista

**Comment by RedSilencer** [comment]
I agree that's a pretty good spot for a vista. Stract may complain about visibility issues, however. An alternative would be something like an adjacent test chamber's outside wall visible through a busted-down or incomplete wall.
